config/cmake/HDF5PluginMacros.cmake

```diff
@@ -3,114 +3,72 @@
 #-------------------------------------------------------------------------------
 macro (EXTERNAL_PLUGIN_LIBRARY compress_type)
   if (${compress_type} MATCHES "GIT")
-    EXTERNALPROJECT_ADD (PLUGIN
+    FetchContent_Declare (PLUGIN
         GIT_REPOSITORY ${PLUGIN_URL}
         GIT_TAG ${PLUGIN_BRANCH}
-        LIST_SEPARATOR |
-        INSTALL_COMMAND ""
-        CMAKE_ARGS
-            -DUSE_SHARED_LIBS:BOOL=ON
```

Comment:
These settings are now reflected in the config/cmake/HDF5PluginsCache.cmake file and the target commands added below.